flink-streaming-java/src/main/java/org/apache/flink/streaming/runtime/operators/windowing/WindowOperator.java

```diff
@@ -131,8 +131,10 @@
 	protected final TypeSerializer<W> windowSerializer;
 
 	/**
-	 * The allowed lateness for elements.
-	 * By default this is set to {@code 0}.
+	 * The allowed lateness for elements. This is used for:
+	 * <li> Deciding if an element should be dropped from a window due to lateness. </li>
```

Comment:
I think you need an `<ul>` around this whole list. And the closing `</li>` is not used in Javadoc, I think.